README.md

```diff
@@ -1,6 +1,17 @@
 # Budget Rails app
 
 This project is for developing a budget app using rails
+
+- [Live Demo - Heroku](https://intense-harbor-49660.herokuapp.com/)

```

Comment:
I was able to access your app once and it developed a fault. please upload the recent commit or check for the error.